packages/features/settings/BookerLayoutSelector.tsx

```diff
@@ -108,9 +116,11 @@ const BookerLayoutFields = ({
   showUserSettings,
   isDark,
   isOuterBorder,
+  user,
+  isUserLoading,
 }: BookerLayoutFieldsProps) => {
   const { t } = useLocale();
-  const { isPending: isUserLoading, data: user } = useMeQuery();
+  // const { isPending: isUserLoading, data: user } = useMeQuery();
```

Comment:
removed this query. now sending from the parent as prop.

Reply:
https://github.com/calcom/cal.com/pull/16711/commits/f28430e29737e02eb9c144e4910b257c69c750ef

Reply:
let's remove the comment if we dont need this anymore

Reply:
https://github.com/calcom/cal.com/commit/7300c9de125390dc7a87d7a9162cc714508097bf